posthog/models/project.py

```diff
@@ -50,3 +56,7 @@ def __str__(self):
         return str(self.pk)
 
     __repr__ = sane_repr("id", "name")
+
+    @cached_property
+    def passthrough_team(self) -> "Team":
```

Comment:
Should we consider having a "primary_team" field instead of using this hack?

Worried that this has the potential to be leaky - once we get one project with multiple teams, this field might point to another project's team.

Reply:
There is actually a guarantee in place that `Team.objects.get(id=project.id)` _always_ returns the projects' first team – even for projects with multiple environments. That's because at this point all Project and Team objects use the same source of IDs: `Team.objects.increment_id_sequence()`.

Reply:
I see that it works now, I guess I'm assuming that it won't always be true (or it could always be true and we just allow team ids to have big gaps), so it would be better to not build logic in that depends on that?

Not awful to migrate later when we're ready to diverge, so nbd either way, but just a thing.

Reply:
The idea is that it will always be true, because it makes migrating things easier in the beginning. E.g. in ingestion, where events will continue to be Team-specific, but group definitions will become Project-specific